doc/sphinx/addendum/program.rst

```diff
@@ -335,18 +335,12 @@ optional tactic is replaced by the default one if not specified.
 
 .. flag:: Hide Obligations
 
+   .. deprecated:: 8.12
+
    Controls whether obligations appearing in the
    term should be hidden as implicit arguments of the special
    constantProgram.Tactics.obligation.
```

Comment:
Could use a space while you're modifying this area btw

Reply:
Fixed thanks.